aggregator/list.go

```diff
@@ -61,22 +62,17 @@ func newMetricListMetrics(scope tally.Scope) metricListMetrics {
 		flushCollected:      flushScope.Counter("collected"),
 		flushSuccess:        flushScope.Counter("success"),
 		flushErrors:         flushScope.Counter("errors"),
-		flushDuration:       flushScope.Timer("duration"),
 		flushStale:          flushScope.Counter("stale"),
-		flushBeforeDuration: scope.Timer("flush-before.duration"),
 		flushDiscarded:      flushScope.Counter("discarded"),
+		flush:               scope.Counter("flush"),
```

Comment:
How is this different to the total of success and errors? Also a bit weird to have a count with name flush and a scope with the same prefix. Why aren't these all under a flush scope?

Reply:
Can get rid of the flush counter and use `flushSuccess` and `flushErrors` instead. Sure can consolidate the metric scope too.